protoactor/protos_pb2.py

```diff
@@ -0,0 +1,267 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
I'm not sure if this file needs to be committed. It is generated when you run `protoc`, if it's needed for compilation of the whole module then it could be added as a step in travis.